pic-sure-resources/pic-sure-resource-api/src/main/java/edu/harvard/dbmi/avillach/service/ResourceWebClient.java

```diff
@@ -240,7 +240,13 @@ public Response queryResult(String rsURL, String queryId, QueryRequest queryRequ
                 HttpClientUtil.composeURL(rsURL, pathName), createHeaders(queryRequest.getResourceCredentials()), body
             );
 
-            String content = httpClientUtil.readObjectFromResponse(resourcesResponse);
+            byte[] content = httpClientUtil.readBytesFromResponse(resourcesResponse);
+            FileOutputStream fos = new FileOutputStream("/tmp/test.avro");
```

Comment:
Do we want this hardcoded in this file? If yes, can we move it to the top of the class/file?

Reply:
Bahhh, this was for debugging. I'll remove it